### ISSUE TITLE
Layer control announces "Layers link" in NVDA

### ISSUE THREAD

**Issue by prushforth** [Maps4HTML/Web-Map-Custom-Element#725]
When focus falls on the Layer control, NVDA announces "Layers - link".  By comparison, I think it should be a similar announcement to when focus falls on a zoom in / out button, i.e. shouldn't the layer control be a button, or perhaps "dialog" might be more appropriate, but definitely not "link" (IMHO).  @Malvoz  wdyt?

**Comment by Malvoz** [comment]
This was fixed upstream in https://github.com/Leaflet/Leaflet/pull/7850 and should no longer be the case, unless the `mapml-viewer.js` in use is of an older version of course... Are you testing a map served at an NRCan resource perhaps?

**Comment by prushforth** [comment]
Shoot, my bad, sorry to bug you.  Yes, I was looking at an old build on Leaflet 1.7.1.  I'm happy to say that we have succeeded to upgrade to Leaflet 1.9.3, so some of the upstream issues (like this one) are auto-solved, thanks to Leaflet contributors. 👍
